docs/install.rst

```diff
@@ -8,9 +8,9 @@ Install version 3 with :pip:ref:`pip install`::
 
     pip install django-autocomplete-light
 
-Or, install the dev version with git::
+Or, install the dev version from Github::
 
-    pip install -e git+https://github.com/yourlabs/django-autocomplete-light.git#egg=django-autocomplete-light
+    pip install https://github.com/yourlabs/django-autocomplete-light/archive/master.zip
```

Comment:
On Wed, Mar 16, 2016 at 11:37 AM, Andy Baker <notifications@github.com>
wrote:

> Yep. No harm having both. I just wanted to correct the implication that -e
> was the only way to install from Github (I thought this to be the case for
> a while myself!)
>
> Do you want another PR or shall I leave it to you?
>
> It would be great if you updated the pr !

Thanks !